packages/adblocker/test/engine/engine.test.ts

```diff
@@ -139,19 +139,18 @@ function buildResourcesFromRequests(filters: NetworkFilter[]): Resources {
       const redirect = filter.redirect;
 
       // Guess resource type
-      const contentType = 'application/javascript';
       if (redirect.endsWith('.gif')) {
         resources.redirects.push({
           names: [redirect],
           content: '',
-          contentType,
+          contentType: 'image/gif;base64',
```

Comment:
I guess in the distribution you should not have `base64` in the content type as it is indicated by the `encoding`